neural_modelling/src/neuron/models/neuron_model_lif_impl.h

```diff
@@ -50,11 +80,36 @@ typedef struct neuron_t {
 
     //! refractory time of neuron [timesteps]
     int32_t  T_refract;
-} neuron_t;
+};
+
+static inline int32_t lif_ceil(REAL value) {
```

Comment:
The "lif" bit is there because it is in the lif file; it avoid name clash should there be another version elsewhere.  I guess this would move if needed elsewhere though of course!  I have added "_accum" to the end and some docs.